stl/inc/chrono

```diff
@@ -6271,3 +6272,4 @@ _STL_RESTORE_CLANG_WARNINGS
 #pragma pack(pop)
 #endif // _STL_COMPILER_PREPROCESSOR
 #endif // _CHRONO_
+

```

Comment:
This will fail validation due to two newlines at the end of the file. :scream_cat: